homeassistant/components/binary_sensor/xiaomi_aqara.py

```diff
@@ -203,26 +223,20 @@ def parse_data(self, data, raw_data):
                     self._unsub_set_no_motion()
                 self._unsub_set_no_motion = async_call_later(
                     self._hass,
-                    180,
+                    120,
                     self._async_set_no_motion
                 )
-            else:
-                self._should_poll = True
-                if self.entity_id is not None:
-                    self._hass.bus.fire('xiaomi_aqara.motion', {
-                        'entity_id': self.entity_id
-                    })
+
+            if self.entity_id is not None:
+                self._hass.bus.fire('xiaomi_aqara.motion', {
+                    'entity_id': self.entity_id
+                })
 
             self._no_motion_since = 0
             if self._state:
                 return False
             self._state = True
             return True
-        if value == NO_MOTION:
```

Comment:
This is dead code. There isn't a "{'data': {'status':'no_motion'}}" response (anymore?).